.github/workflows/go.yml

```diff
@@ -4,34 +4,25 @@
 name: Go
 
 on:
-  pull_request_target:
-    branches: [ "main" ]
+  push:
+    branches:
+      - '*'
+  
 
 env:
   TURSO_API_TOKEN: ${{ secrets.TURSOCI_TOKEN }}
 
 jobs:
-
-  authorize:
```

Comment:
@athoscouto we need this to be able to run CI for external contributions. Or have you found another way of doing that safely?